test/edu-benefits/pages/schoolSelection.unit.spec.jsx

```diff
@@ -52,6 +52,40 @@ describe('Edu 1990e schoolSelection', () => {
     expect(Array.from(formDOM.querySelectorAll('.usa-input-error'))).to.be.empty;
     expect(onSubmit.called).to.be.true;
   });
+
+  it('should require educationType if specified', () => {
+    const { schema: schemaIfEducationType, uiSchema: uiSchemaIfEducationType } = formConfig(fullSchema1990e, {
+      fields: [
+        'educationProgram',
+        'educationObjective',
+        'nonVaAssistance',
+        'civilianBenefitsAssistance'
+      ],
+      required: ['educationType']
+    });
+
+    const onSubmit = sinon.spy();
+    const form = ReactTestUtils.renderIntoDocument(
+      <DefinitionTester
+          schema={schemaIfEducationType}
+          onSubmit={onSubmit}
+          data={{}}
+          definitions={formConfig.defaultDefinitions}
+          uiSchema={uiSchemaIfEducationType}/>
+    );
+    const formDOM = findDOMNode(form);
+    const find = formDOM.querySelector.bind(formDOM);
+
+    ReactTestUtils.Simulate.change(find('#root_educationProgram_educationType'), {
```

Comment:
How about verifying that
  - an error is rendered
  - onSubmit isn't called
and then providing the education type